src/components/NumericStepper/__tests__/NumericStepper.spec.js

```diff
@@ -11,6 +11,14 @@ describe('<NumericStepper> component', () => {
         jest.resetAllMocks();
     });
 
+    const decreaseClick = () => {
+        wrapper.find('StepperButton').at(0).prop('onClick')();
+    };
+
+    const increaseClick = () => {
+        wrapper.find('StepperButton').at(1).prop('onClick')();
+    };
```

Comment:
nice! It is easier to read the tests when they call functions like this with meaningful name. Nice improvement.

I also didn't know you can call onClick like this. We normally use `simulate('click')`. Good to learn something new each time :-)

Reply:
Yeah, I've read somewhere that `simulate` might be deprecated soon.